connectors/src/connectors/slack/index.ts

```diff
@@ -68,7 +68,7 @@ export async function createSlackConnector(
         { transaction: t }
       );
 
-      launchSlackSyncWorkflow(connector.id.toString());
+      await launchSlackSyncWorkflow(connector.id.toString());
```

Comment:
Please handle in a separate PR along with the implementation of deleteConnector.